### ISSUE TITLE
Travis fails with hostlist

### ISSUE THREAD

**Issue by dongahn** [flux-framework/flux-sched#76]
I'm getting Travis CI build failures. I initially thought this was because of my changes in the waitjob_fix branch of my fork. But then when I sync'ed the master of my fork with the upstream sched master, I notice I'm still getting the same failure. Travis CI  didn't complained about @lipari's latest PR so I'm kind of baffled. 

```
ok 3 - resoure file readable
loading RDL: /home/travis/build/dongahn/flux-sched/rdl/RDL.lua:30: module 'hostlist' not found:
	no field package.preload['hostlist']
	no file '/usr/local/share/lua/5.1//hostlist.lua'
	no file '/usr/local/share/lua/5.1//hostlist/init.lua'
	no file '/home/travis/.luarocks/share/lua/5.1//hostlist.lua'
	no file '/home/travis/.luarocks/share/lua/5.1//hostlist/init.lua'
	no file '/usr/share/lua/5.1//hostlist.lua'
	no file '/usr/share/lua/5.1//hostlist/init.lua'
	no file './hostlist.lua'
	no file '/usr/local/share/lua/5.1/hostlist.lua'
	no file '/usr/local/share/lua/5.1/hostlist/init.lua'
	no file '/usr/local/lib/lua/5.1/hostlist.lua'
	no file '/usr/local/lib/lua/5.1/hostlist/init.lua'
	no file '/usr/share/lua/5.1/hostlist.lua'
	no file '/usr/share/lua/5.1/hostlist/init.lua'
	no file '/home/travis/build/dongahn/flux-sched/rdl/hostlist.lua'
	no file '/home/travis/flux-core/src/bindings/lua/hostlist.lua'
	no file './hostlist.lua'
	no file '/usr/local/share/lua/5.1/hostlist.lua'
	no file '/usr/local/share/lua/5.1/hostlist/init.lua'
	no file '/usr/local/lib/lua/5.1/hostlist.lua'
	no file '/usr/local/lib/lua/5.1/hostlist/init.lua'
	no file '/usr/share/lua/5.1/hostlist.lua'
	no file '/usr/share/lua/5.1/hostlist/init.lua'
	no file '/usr/local/lib/lua/5.1//hostlist.so'
	no file '/home/travis/.luarocks/lib/lua/5.1//hostlist.so'
	no file './hostlist.so'
	no file '/usr/local/lib/lua/5.1/hostlist.so'
	no file '/usr/lib/x86_64-linux-gnu/lua/5.1/hostlist.so'
	no file '/usr/lib/lua/5.1/hostlist.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file '/home/travis/build/dongahn/flux-sched/rdl/hostlist.so'
	no file '/home/travis/flux-core/src/bindings/lua/.libs/hostlist.so'
	no file './hostlist.so'
	no file '/usr/local/lib/lua/5.1/hostlist.so'
	no file '/usr/lib/x86_64-linux-gnu/lua/5.1/hostlist.so'
	no file '/usr/lib/lua/5.1/hostlist.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
not ok 4 - resource generation took: 0.001116
#   Failed test 'resource generation took: 0.001116'
#   at tresrc.c line 130.
# Looks like you planned 13 tests but ran 4.
# Looks like you failed 1 test of 4 run.
make[3]: *** [check] Error 1
make[3]: Leaving directory `/home/travis/build/dongahn/flux-sched/resrc/test'
make[2]: *** [check] Error 2
make[2]: Leaving directory `/home/travis/build/dongahn/flux-sched/resrc'
make[1]: *** [resrc] Error 2
make[1]: Leaving directory `/home/travis/build/dongahn/flux-sched'
make: *** [check] Error 2

The command "./config $HOME/flux-core && make check" exited with 2.
store build cache
changes detected, packing new archive
uploading archive

Done. Your build exited with 1.
```

**Comment by grondo** [comment]
Ah, this is another case where I failed to change a `require 'hostlist'` to `require 'flux.hostlist'`, but things continued to work because the travis cache still had lua-hostlist installed. When you build on a new branch you get fresh caches which is why this was only caught now. Sorry!

You can fix by replacing occurrences of `require 'hostlist'` with `require 'flux.hostlist'` -- or I could do it if you want.

**Comment by dongahn** [comment]
Thanks. I will take care.

**Comment by grondo** [comment]
Ok, thanks. Before I saw your comment I pushed up to my branch at least part of fix, you can check it out [here](https://github.com/grondo/flux-sched/commit/c7cd2040662d049cc078f15822a92dbcd189db5a)

Thanks!

**Comment by dongahn** [comment]
This is fixed with commit 07552b6 from PR#77.
